src/views/pacbio/PacbioLibraries.vue

```diff
@@ -143,7 +142,7 @@ export default {
     // Get all the libraries
     // Provider function used by the bootstrap-vue table component
     async provider() {
-      this.items = await this.getMaterial(consts.MAT_TYPE_LIBRARIES)
+      this.items = await this.setLibraries()
```

Comment:
Same here as below comment

Reply:
Modified to use getter for libraries. Removed clear pre filter. Removed unneeded getters.